### ISSUE TITLE
Bundle install: rbglib_datetime.c:164:24: error: ‘G_TYPE_TIME_ZONE’ undeclared

### ISSUE THREAD

**Issue by i2097i** [ruby-gnome2/ruby-gnome2#1012]
When running running bundle install on TravisCI, glib2 hits a gem build error. Here is the specific portion of the log:

```
Fetching gem metadata from https://rubygems.org/..........
Fetching version metadata from https://rubygems.org/.
Resolving dependencies...
Installing pkg-config 1.1.7
Using bundler 1.14.6
Installing rake 10.5.0
Installing diff-lcs 1.3
Installing mini_portile2 2.1.0
Installing wannabe_bool 0.6.0
Installing rspec-support 3.5.0
Installing glib2 3.1.1 with native extensions
Installing cairo 1.15.5 with native extensions
Installing nokogiri 1.7.1 with native extensions
Installing rspec-core 3.5.4
Installing rspec-expectations 3.5.0
Installing rspec-mocks 3.5.0
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

...

rbglib_datetime.c: In function ‘rg_s_now’:
rbglib_datetime.c:73:36: error: ‘G_TYPE_TIME_ZONE’ undeclared (first use in this
function)
rbglib_datetime.c:73:36: note: each undeclared identifier is reported only once
for each function it appears in
rbglib_datetime.c: In function ‘rg_initialize’:
rbglib_datetime.c:164:24: error: ‘G_TYPE_TIME_ZONE’ undeclared (first use in
this function)
make: *** [rbglib_datetime.o] Error 1
make failed, exit code 2
Gem files will remain installed in
/home/travis/.rvm/gems/ruby-2.3.0-dev/gems/glib2-3.1.1 for inspection.
Results logged to
/home/travis/.rvm/gems/ruby-2.3.0-dev/extensions/x86_64-linux/2.3.0/glib2-3.1.1/gem_make.out
An error occurred while installing glib2 (3.1.1), and Bundler cannot
continue.
Make sure that `gem install glib2 -v '3.1.1'` succeeds before bundling.
```

Here is a [link](https://travis-ci.org/i2097i/gluby-tk) to the entire build log

**Comment by cedlemo** [comment]
Just to be sure, I checked that the macro `G_TYPE_TIME_ZONE` still exists : 

https://developer.gnome.org/gobject/unstable/gobject-Boxed-Types.html#G-TYPE-TIME-ZONE:CAPS

did you installed all the dependencies ?

**Comment by kou** [comment]
Use `dist: trusty` in `.travis.yml`: https://docs.travis-ci.com/user/trusty-ci-environment/

Ubuntu precise is old.

**Comment by kou** [comment]
Ah, sorry. We should support GLib 2.32.

I've fixed at a40422dd266e62c882944f7aa34539cae1376fbf.
